tests/tests/test-ethers.ts

```diff
@@ -0,0 +1,57 @@
+import { createAndFinalizeBlock, describeWithMoonbeam } from "./util";
+import { HttpProvider } from "web3-core";
+import { expect } from "chai";
+import { ethers } from "ethers";
+import { TEST_CONTRACT_ABI, TEST_CONTRACT_BYTECODE } from "./constants";
+
+describeWithMoonbeam("Moonbeam RPC (Ethers.js)", `simple-specs.json`, (context) => {
+  const GENESIS_ACCOUNT_PRIVATE_KEY =
+    "0x99B3C12287537E38C90A9219D4CB074A89A16E9CDB20BF85728EBD97C343E342";
+
+  it("get network ids", async function () {
+    // Providers
+    let prov = context.web3.currentProvider as HttpProvider;
+    const provider = new ethers.providers.JsonRpcProvider(prov.host);
```

Comment:
Looks good.
If we are planning to have more ethers test, we should move it to the providers list in the describeWithMoonbeam

Reply:
I don't have more ethers tests in mind for right now but I'll definitely move it to describeWithMoonbeam when we continue ethers testing